PACKAGES.md

```diff
@@ -0,0 +1,99 @@
+# Fortran-lang.org package index
+
+
+## Package criteria
+
+The following criteria are required of packages to be indexed:
+
+- __Relevance__: the package must be primarily implemented in Fortran or provide
+a complete Fortran interface to an existing package or be purposed solely towards
+software development in Fortran.
+
+- __Maturity__: the primary functionality of the package shall be implemented.
+No prototype, testing or partially complete packages will be accepted.
+If the package is hosted on github or similar, it should have at least 5 'stars'.
+
+- __Availability__: the package source shall be freely available for browsing online
+or cloning or downloading
+
+- __Open source__: the package shall be licensed under an appropriate [open-source license](https://opensource.org/licenses)
+with the license file clearly included with the source code
+
+- __Uniqueness__: the package shall not be a fork or minor revision of existing packages
+
+- __README__: the package shall have some form of README or landing-page clearly
+stating the package purpose and functionality. This should also contain information
+on the package dependencies and the steps required to build and run.
+
+
+The following criteria are not required but are recommended:
+
+- __Documentation__: any form of written documentation aimed at users of the package. Ideally
+this should cover:
+  - Supported / tested compilers
+  - Dependencies
+  - Build and install process
+  - Modules contained within the package
+  - Procedures made available and their interfaces
+  - Example code
+
+- __Contributing__: details on how users may submit issues and contribute to the development of the
+package
+
+- __Tests__: any form of executable test(s) that can be used to verify the functionality of the package
+
+- __Portability__: no non-standard language extensions or proprietary dependencies
+
+- __FPM__: support installation by the Fortran Package Manager [fpm](https://github.com/fortran-lang/fpm)
+
+
+## Process for adding packages
+
+1. Users should confirm that their project meets the minimum requirements for listing in the 
+Fortran-lang package index, as written in this document
+
+2. Users should open an issue using the 'Package index request' issue template
```

Comment:
Should we encourage going straight to PR here rather than opening an issue? Same discussion and approval can be done in the PR discussion thread, and it would allow people to contribute directly.

Reply:
> Should we encourage going straight to PR here rather than opening an issue? Same discussion and approval can be done in the PR discussion thread, and it would allow people to contribute directly.

Actually yeah this is a better idea. Simpler process.

Reply:
Me too (go right for the pull request).